src/cloud/cloud_init.rs

```diff
@@ -0,0 +1,41 @@
+pub fn generate_cloud_init_config(password: &str, port: u16) -> String {
+    let cloud_config = serde_json::json!({
+      "runcmd": ["curl https://i.jpillora.com/chisel! | bash", "systemctl enable --now chisel"],
```

Comment:
sometimes I wish we had our own image... but maintaining images for all of these stupid cloud providers is.. yeah